### DISCUSSION TITLE
ACE Editor Save button

### DISCUSSION THREAD

**Discussion by Greentesco** [ajaxorg/ace#5332] [General]
Hello, thank you so much for your product, I would like to clarify how the function of saving settings in the editor is implemented. At the moment, when selecting the editor "ACE Editor" in the file manager, there are theme settings for this editor when selecting the desired theme and settings are applied, but after closing the editor and opening it again, the settings are set to standard. Is it possible to add a save button? or the save script works but saves the configuration file on the server. It would be very nice if you added a Save button Thanks again for your product

**Comment by nightwing** [comment]
Hi ace editor is just a library used in other products to provide code editing. It does not handle the theme settings, so you need to report this bug to the product which you are using.

**Comment by ldijkman** [comment]
Hello

what would be a good way to store settings
coockies is only for one browser?


Luberth
https://ldijkman.github.io/Ace_Seventh_Heaven/Seventh_Heaven.html
